tutorials/6_Advanced_ChangeInputs.md

```diff
@@ -44,3 +44,20 @@ Adding New Input Data
 
 The input data for REMIND are generated by using the R-libraries *mrremind* (https://github.com/pik-piam/mrremind) and *madrat* (https://github.com/pik-piam/madrat). While *mrremind* contains all calculations tailored to REMIND-input-data, the package *madrat* provides the general wrapper functions and helpful tools. For further information read the vignettes part of the R-packages.
 
+How to Update Input Data
+========================
+
+
+1. Check for the current input data revision number in this cluster folder: `/p/projects/rd3mod/inputdata/cache`
+2. Clone the repo `https://gitlab.pik-potsdam.de/REMIND/preprocessing-remind` to any location on the cluster and edit the `start.R` file by inserting the next revision number. If an old revision number is used, the input data will not be recalculated.
```

Comment:
maybe mention that for new regional resolution it will be recalculated based on the existing cache information

Reply:
has been updated